### PR TITLE
[6_0_X][TIMOB-24429] Only disable HW-acceleration for borderView below Jelly Bean

### PR DESCRIPTION
**Pull request by garymathews**
- Cherry-pick of https://github.com/appcelerator/titanium_mobile/pull/8851

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-24429)

### CODE REVIEW

**Review by antw12 — APPROVED** [review]
CR & FT PASS 👍

**Comment by lokeshchdhry** [comment]
@garymathews , I don't see any map on below lollypop devices. I just see the parent view.
Runs fine on above lollypop devices.

**Comment by garymathews** [comment]
@lokeshchdhry Updated PR

File: android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java [hunk 1]
```diff
@@ -1324,7 +1325,7 @@ private void initializeBorder(KrollDict d, Integer bgColor)
 					if (radiusDim != null) {
 						radius = (float) radiusDim.getPixels(getNativeView());
 					}
-					if (radius > 0f && HONEYCOMB_OR_GREATER) {
+					if (radius > 0f && HONEYCOMB_OR_GREATER && LOWER_THAN_JELLYBEAN) {
```
Comment (lokeshchdhry) [inline android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java]: Should it be `JELLY_BEAN` ?
I am getting build failure with:
```
[javac] /Users/lchoudhary/Desktop/repositories/titanium_mobile/android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java:91: error: cannot find symbol
[javac] 	private static final boolean LOWER_THAN_JELLYBEAN = (Build.VERSION.SDK_INT < Build.VERSION_CODES.JELLYBEAN);
    [javac] 	                                                                                                ^
    [javac]   symbol:   variable JELLYBEAN

[javac]   location: class VERSION_CODES
```

File: android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java [hunk 2]
```diff
@@ -80,6 +80,7 @@
 
 	private static final boolean HONEYCOMB_OR_GREATER = (Build.VERSION.SDK_INT >= 11);
 	private static final boolean LOLLIPOP_OR_GREATER = (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP);
+	private static final boolean LOWER_THAN_JELLYBEAN = (Build.VERSION.SDK_INT < Build.VERSION_CODES.JELLYBEAN);
```
Comment (lokeshchdhry) [inline android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java]: Should it be `JELLY_BEAN` ?
I am getting build failure with:
```
[javac] /Users/lchoudhary/Desktop/repositories/titanium_mobile/android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java:91: error: cannot find symbol
[javac] 	private static final boolean LOWER_THAN_JELLYBEAN = (Build.VERSION.SDK_INT < Build.VERSION_CODES.JELLYBEAN);
    [javac] 	                                                                                                ^
    [javac]   symbol:   variable JELLYBEAN

[javac]   location: class VERSION_CODES
```

**Comment by lokeshchdhry** [comment]
FR Passed.

Map shows successfully if the parent view has `borderRadius`.
Tested on above & below android lollipop devices.

Appc Studio : 4.8.1.201612050850
SDK Version : Locally built 6.0.3
Mac OS Version : 10.12.3
Xcode Version : Xcode 8.2.1 Build version 8C1002
Appc CLI AND Appc NPM : {"NPM":"4.2.8","CLI":"6.1.0"}
Ti CLI : 5.0.11
Alloy : 1.9.5
Node : v4.6.0

**Pull request merged**